conf/systemd.service

```diff
@@ -1,16 +1,14 @@
 [Unit]
 Description=Home Assistant
-After=network-online.target
+After=network.target
 
 [Service]
 Type=simple
 User=__APP__
-WorkingDirectory=__PATH__
-ExecStart=/opt/yunohost/__APP__/bin/hass --config "__PATH__" --verbose
-StandardOutput=append:/var/log/__APP__.log
-StandardError=inherit
```

Comment:
Why remove these though ?

Reply:
to avoid duplicate log entries: the app already redirect service log